jsonpickle/util.py

```diff
@@ -481,8 +486,14 @@ def importable_name(cls: Union[Type, Callable[..., Any]]) -> str:
     True
     >>> importable_name(AttributeError) == 'builtins.AttributeError'
     True
+    >>> importable_name(type(argparse.ArgumentParser().add_argument)) == 'types.MethodType'
+    True
 
     """
+    types_importable_name = _TYPES_IMPORTABLE_NAMES.get(id(cls))
```

Comment:
Sorry if this is a stupid question (I just got back home from the airport and am super jet lagged) but if two objects of different types have the [same id because of non-overlapping lifetimes](https://docs.python.org/3/library/functions.html#id), could this break, especially if someone is dynamically adding types?

Reply:
Do you think if it makes sense to use the type as the key?